CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(ENABLE_GOOGLETEST_INSTALL "Skip installation" OFF )
```

Comment:
what other lines have stray spaces? Just change
`option(ENABLE_GOOGLETEST_INSTALL "Skip installation" OFF )`
to
`option(ENABLE_GOOGLETEST_INSTALL "Skip installation" OFF)`